source/operation.ts

```diff
@@ -81,7 +72,7 @@ export type Operation =
  * @param  {Object} data Entity data to use for creation
  * @return {Object}      API operation
  */
-export function createOperation(type: string, data: any): CreateOperation {
+export function create(type: string, data: any): CreateOperation {
```

Comment:
Even though they probably are not ment to be used outside this project, must we not wait with changing function names until it's time to make a major version release?

Reply:
they were not exposed publicly from index.ts, so only way to have used them would be if someone imported them directly from `@ftrack/api/dist/operation`, which I don't think is supported behavior.

Reply:
I think I'm just overly cautious.